coding_schemes/demog_schemes.csv

```diff
@@ -0,0 +1,8 @@
+Clan
```

Comment:
Not sure if this should be committed

Reply:
Why?

Reply:
We don't commit the CSVs - they're not the canonical form of the coding schemes - the canonical form is the JSON files. Please could you remove this.